.circleci/merge_pr.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
As far as I understand, we run CI jobs against PRs that are merged in the base. That is to prevent CI runs go green when in reality they create problems after they are merged. So it's another safety measure.